### PR TITLE
PMP corefinement: Handle inconsistency with open mesh as input

### PR DESCRIPTION
**Pull request by sloriot**
Return false for such a case:
Screenshot from 2022-09-08 19-05-31

### CODE REVIEW

File: Polygon_mesh_processing/test/Polygon_mesh_processing/test_corefinement_bool_op.cmd [hunk 1]
```diff
@@ -1 +1,2 @@
 ${CGAL_DATA_DIR}/meshes/elephant.off ${CGAL_DATA_DIR}/meshes/sphere.off ALL 1 1 1 1
+${CGAL_DATA_DIR}/meshes/open_cube.off data-coref/incompatible_with_open_cube.off ALL 0 0 0 0
```
Comment (lrineau) [inline Polygon_mesh_processing/test/Polygon_mesh_processing/test_corefinement_bool_op.cmd:2]: Does that work correctly to have multiple lines in `.cmd` files? Even with ctest?

Comment (sloriot) [reply Polygon_mesh_processing/test/Polygon_mesh_processing/test_corefinement_bool_op.cmd:2]: the test handle them correctly (there is a for loop taking argc by tuples)

Comment (lrineau) [reply Polygon_mesh_processing/test/Polygon_mesh_processing/test_corefinement_bool_op.cmd:2]: I see that ctest is launching:
> .../test/Polygon_mesh_processing/test_corefinement_bool_op "/home/lrineau/Git/cgal/Data/data/meshes/elephant.off" "/home/lrineau/Git/cgal/Data/data/meshes/sphere.off" "ALL" "1" "1" "1" "1" "/home/lrineau/Git/cgal/Data/data/meshes/open_cube.off" "data-coref/incompatible_with_open_cube.off" "ALL" "0" "0" "0" "0"

but it does not seems that the `main` of the test can handle 16 arguments.

Comment (sloriot) [reply Polygon_mesh_processing/test/Polygon_mesh_processing/test_corefinement_bool_op.cmd:2]: You are right, this mechanism is implemented only in test_corefine.

**Comment by sloriot** [comment]
Successfully tested in CGAL-5.6-Ic-79

**Pull request merged**